core/orch_test.go

```diff
@@ -224,15 +224,23 @@ func TestSelectTranscoder(t *testing.T) {
 
 	assert.NotNil(m.liveTranscoders[strm])
 	assert.NotNil(m.liveTranscoders[strm2])
-	assert.Len(m.remoteTranscoders, 9)
+	assert.Len(m.remoteTranscoders, 2)
 
 	// assert transcoder is returned from selectTranscoder and removed from list
 	t1 := m.liveTranscoders[strm]
 	t2 := m.liveTranscoders[strm2]
 	currentTranscoder := m.selectTranscoder()
-	assert.Equal(currentTranscoder, t2)
+	assert.Equal(t2, currentTranscoder)
+	assert.Equal(1, t2.load)
 	assert.NotNil(m.liveTranscoders[strm])
-	assert.Len(m.remoteTranscoders, 8)
+	assert.Len(m.remoteTranscoders, 2)
+
+	// assert transcoder with less load selected
+	currentTranscoder2 := m.selectTranscoder()
+	assert.Equal(t1, currentTranscoder2)
+	assert.Equal(1, t1.load)
+	m.completeTranscoders(t1)
+	assert.Equal(0, t1.load)
```

Comment:
Would be good to add another test case that attempts to select transcoders when all are at capacity - `selectTranscoder` should return nil.